components/backdrop/test/backdrop.visual-diff.js

```diff
@@ -0,0 +1,32 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-backdrop', () => {
+
+	const visualDiff = new VisualDiff('backdrop', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.emulateMediaFeatures([{ name: 'prefers-reduced-motion', value: 'reduce' }]);
```

Comment:
Probably don't want everyone to have to set this in all their tests (they'll forget, or do it inconsistently)... is there an easier way we could expose this for everyone?

Reply:
🤔 of some options but they all have issues.  Perhaps lets discuss tomorrow if you have time.  :)